.github/workflows/test.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Use Node.js
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v2
         with:
           node-version: 16.x
       - id: yarn-cache-dir-path
```

Comment:
Could you also make use of the use `cache: yarn` option and get rid of the following 2 steps about yarn caching

Reply:
yes, on it.